src/api/rest/v1/ds/controller.ts

```diff
@@ -1,3 +1,4 @@
+const _ = require('lodash')
```

Comment:
I recommend to destructure the needed functions rather than importing the whole library. Not a big deal on backend but very much needed on frontend to benefit from the tree shaking and prevent sending the whole library to the client when only one function is used. So, a good habit to have.

Also, we should stay consistent with the import vs require statements

```typescript
import { merge } from 'lodash'
```

Reply:
Noted for future